app/views/common_parameters/_parent_parameter.html.erb

```diff
@@ -7,7 +7,7 @@
           <%= content_tag(:input, '', :class => "form-control", :disabled => true,  :value => _('Global'), :type=> :text ) %>
         </td>
         <td class="col-md-2">
-          <%= text_field_tag("parent_parameter_#{i}_key", '', :class => "form-control", :placeholder => parameter.name) %>
+          <%= text_field_tag("parent_parameter_#{i}_key", '', :class => "form-control", :value => parameter.name) %>
```

Comment:
Yeah, it ought to be possible to override a parent or global parameter with a blank value, but it would need a significant change to the form - probably to work more like smart class parameters, where you only see the current value outside of an input box, click Override to get an input box and that causes the new parameter to be created.  The way this works currently, showing the parent value in an input box probably means it'd be overly messy to support blank overridden parameters.